.github/workflows/build.yml

```
@@ -1,6 +1,8 @@
 name: Build
 
-on: [push, pull_request]
+on: [push]
```

I think we still want this to run even on pull requests.  In the special case that the pull request is coming from this same repository, then that does unfortunately result in the workflow running twice.   But I don't think that is a big deal.

---

sorry, I'll revert that. I was trying to figure out why the PR wasn't triggering the CI, but I think it has something to do with originating from my fork. The Ci runs (and passes) on my fork...

The double triggered workflow could be avoided by 
```suggestion
on:
  push:
  pull_request:
    types: [opened, reopened]
```
because this leaves out any push event that also triggers a PR synchronize event.